x/dogfood/keeper/impl_sdk.go

```diff
@@ -214,7 +215,12 @@ func (k Keeper) IterateDelegations(
 	panic("unimplemented on this keeper")
 }
 
-func (k Keeper) WriteValidators(ctx sdk.Context) ([]tmtypes.GenesisValidator, error) {
+// WriteValidators returns all the currently active validators. This is called by the export
+// CLI. During this call, ctx.ChainID() is blank, and hence, chainID is passed separately.
+func (k Keeper) WriteValidators(
+	ctx sdk.Context, chainID string,
+) ([]tmtypes.GenesisValidator, error) {
+	fmt.Println("chainID", chainID)
```

Comment:
remove the debug line plz.

Reply:
Reverted the commit. Will fix in a separate PR.